### ISSUE TITLE
cache replacement

### ISSUE THREAD

**Issue by mk228** [ndncomm/nfd-caching-ndnsim#1]
Hi..... I need urgent help.
 If I want to use weight for replacement and weight contains 3 fields. how can calculate this weight in ndn . I have not got anything. for example weight contains frequency , time . sum of 2 fields calculate weight. how write in ndnsim?
thanks a lot

**Comment by schneiderklaus** [comment]
Hi,
can you tell me what exactly you are trying to do? What do you mean with "weight"? What do you mean with "replacement"? The cache replacement policy?

Also, is this related to our implementation or to ndnSIM in general?

**Comment by mk228** [comment]
Hi, I want to cache replacement policy. weight for when cache is full, which one select to evict.
another question, in ndnsim, How can caclculate distance to server????

**Comment by schneiderklaus** [comment]
I think you should post your question about the weight to the NFD mailing list. 

Again: what do you mean with distance to the server? In NDN you usually don't know which server/producer or cache sends you the data. You can measure the RTT between the Interest is sent out and the data packet is received. There may also be a hopcount field that tells you the number of routers that were travelled. 

Have a look at the NFD developer guide: http://named-data.net/publications/techreports/ndn-0021-5-nfd-developer-guide/

**Comment by mk228** [comment]
yes, I want to calculate number of routers that were travelled.how??? can you explain more?
how measure RTT??
